### DISCUSSION TITLE
[Help]: How to set the loop duration?

### DISCUSSION THREAD

**Discussion by MarkusHeinisch** [davidjerleke/embla-carousel#795] [Help] [answered]
### Summary

How can i set the loop duration (lets say 5 seconds) between auto looping? 

### If applicable, which variants of Embla Carousel are relevant to this question?

- [X] embla-carousel **(Core)**
- [ ] embla-carousel-react
- [ ] embla-carousel-vue
- [ ] embla-carousel-svelte
- [X] embla-carousel-autoplay
- [ ] embla-carousel-auto-scroll
- [ ] embla-carousel-solid
- [ ] embla-carousel-auto-height
- [ ] embla-carousel-class-names
- [ ] embla-carousel-docs **(Documentation)**
- [ ] embla-carousel-docs **(Generator)**

**Comment by davidjerleke** [comment, marked as answer]
Hi @MarkusHeinisch,

Thank you for your question. I'm not sure I understand what you mean. Are you looking for the [`delay`](https://www.embla-carousel.com/plugins/autoplay/#delay) option? If yes, here's a [code example](https://www.embla-carousel.com/api/plugins/#constructor-options).

Best,
David

**Reply by MarkusHeinisch** [reply]
Yes, thanks sir.
